### PR TITLE
PMM-8322: Delete artifact from storage

### PR DESCRIPTION
**Pull request by oter**
https://jira.percona.com/browse/PMM-8158

### CODE REVIEW

File: api/managementpb/backup/artifacts.proto [hunk 1]
```diff
@@ -48,6 +48,15 @@ message ListArtifactsResponse {
   repeated Artifact artifacts = 1;
 }
 
+message RemoveArtifactRequest {
+  // Machine-readable artifact ID.
+  string artifact_id = 1;
+  // Removes all the backup files associated with artifact if flag is set.
+  bool complete_removal = 2;
```
Comment (BupycHuk) [inline api/managementpb/backup/artifacts.proto:55]: physical_files?

Comment (BupycHuk) [reply api/managementpb/backup/artifacts.proto:55]: remove_files?

**Review by BupycHuk — APPROVED** [review]
I think complete_removal isn't the best naming

File: api/managementpb/backup/artifacts.proto (hunk 1 above)
Comment (oter) [reply api/managementpb/backup/artifacts.proto]: renamed to `remove_files`

File: api/managementpb/backup/artifacts.proto [hunk 2]
```diff
@@ -48,6 +48,15 @@ message ListArtifactsResponse {
   repeated Artifact artifacts = 1;
 }
 
+message RemoveArtifactRequest {
```
Comment (artemgavrilov) [inline api/managementpb/backup/artifacts.proto:51]: It should be `DeleteArtifactRequest` according to our naming policy.

Comment (oter) [reply api/managementpb/backup/artifacts.proto:51]: done

**Pull request closed without merging**